.gitignore

```diff
@@ -11,6 +11,9 @@ pids
 *.seed
 *.pid.lock
 
+/.idea/
```

Comment:
You can just add `.idea` and it will ignore all the files in it. Also you should add `.vscode` to ignore settings for those who use vs code.